tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import collections
+import os
+
+from lsst.meas.astrom import LoadAstrometryNetObjectsTask, LoadAstrometryNetObjectsConfig
+from lsst.afw import geom, table
+
+from lsst.jointcal import jointcal
+
+arcsec_per_radian = 206265.
```

Comment:
No.  Firstly, no under_scores when you've adopted camelCase (unless you're going to make it all-caps to make clear it's a constant).  But more importantly, you should never need this particular value because it's provided by `lsst.afw.geom.Angle` in a much safer way.